includes/payments/functions.php

```diff
@@ -1189,13 +1189,14 @@ function give_remove_payment_prefix_postfix( $number ) {
  *
  * @param int|Give_Payment $donation    Donation ID or Donation Object.
  * @param bool|array       $format_args Currency Formatting Arguments.
+ * @param string           $type        Currency Formatting Arguments.
```

Comment:
This param needs a better description to what the various types do. I should be able to understand that if I pass `donation` what it does versus `stats` or any other args. Please provide that better description

Reply:
@DevinWalker I will improve this description.